projects/observability/src/shared/graphql/request/handlers/explore/explore-graphql-query-handler.service.ts

```diff
@@ -149,6 +158,11 @@ export interface GraphQlExploreResult {
   [key: string]: GraphQlExploreResultValue;
 }
 
+export enum ExploreQueryContextType {
+  Api = 'API',
```

Comment:
Don't think this is needed - it is required to be identical to the entity type, and isn't used in here.

Reply:
There are so many places where we use this as a string. I've started using this instead.